src/Files.App/Data/Commands/Manager/CommandManager.cs

```diff
@@ -13,7 +13,11 @@ namespace Files.App.Data.Commands
 {
 	internal sealed class CommandManager : ICommandManager
 	{
-		private readonly IGeneralSettingsService settings = Ioc.Default.GetRequiredService<IGeneralSettingsService>();
+		// Dependency injections
+
+		private IGeneralSettingsService GeneralSettingsService { get; } = Ioc.Default.GetRequiredService<IGeneralSettingsService>();
```

Comment:
I don't think it is necessary to convert private readonly fields to properties.

Reply:
As a practice I use private properties for DI instances.

Reply:
A private property internally generates a private readonly field and a getter method, but there is no need to provide a getter method if it is private. It may be better to set a policy in the coding style guidelines.

Reply:
In your opinion what's the best practice? I'd like them to be PascalCase.
I think we can open a ticket for this.

Reply:
I don't have a preference about naming conventions, but ideally I think all properties that can be replaced with private readonly fields should be replaced.

Reply:
I'll keep as is for know.
I'll think about it and if needed I'll open a dedicated issue to address.